src/main/java/org/jenkinsci/plugins/p4/workspace/ManualWorkspaceImpl.java

```diff
@@ -172,12 +172,34 @@ else if (client != null && adjustClientName) {
 	}
 
 
-	private ArrayList<String> getChangeView(WorkspaceSpec workspaceSpec) {
+	private ArrayList<String> getChangeView(IOptionsServer connection, WorkspaceSpec workspaceSpec) throws Exception
+    {
 		ArrayList<String> changeView = new ArrayList<>();
 
 		String view = workspaceSpec.getChangeView();
 		if (view != null) {
 			String specString = getExpand().format(view, true);
+			boolean fromFile = false;
```

Comment:
fromFile variable is never used

Reply:
pls resolve this comment

Reply:
yes, indeed. this was copied from JENKINS-69491 where it was used.
fixed in new commit